editor/src/components/shared/project-components.ts

```diff
@@ -462,6 +474,41 @@ const divComponentGroup = {
   ),
 }
 
+const gridComponentGroup: ComponentDescriptorsForFile = {
+  grid: {
+    properties: {},
+    supportsChildren: true,
+    preferredChildComponents: [],
+    source: defaultComponentDescriptor(),
+    variants: [
+      {
+        insertMenuLabel: 'Grid',
+        elementToInsert: () =>
+          jsxElementWithoutUID(
+            'div',
+            jsxAttributesFromMap({
+              style: jsExpressionValue(
+                {
+                  display: 'grid',
+                  gridTemplateRows: '1fr 1fr 1fr',
+                  gridTemplateColumns: '1fr 1fr 1fr',
+                  gap: 10,
+                  position: 'absolute',
+                  width: 200,
+                  height: 200,
+                },
+                emptyComments,
+              ),
```

Comment:
this should be a shared function for the floating menu and the insert button, so that they remain consistent even if we tweak the default template

Reply:
done in [`8279ecf` (#6512)](https://github.com/concrete-utopia/utopia/pull/6512/commits/8279ecf03f587201a89c4ffb32bb793eb9f3b618)